src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/DebtorsServiceImpl.java

```diff
@@ -95,6 +94,23 @@ public Debtors getDebtors(String transactionId, String companyAccountsId, String
         return new ArrayList<>();
     }
 
+    @Override
+    public void deleteDebtors(String transactionId, String companyAccountsId) throws ServiceException {
```

Comment:
The delete of the resource should be performed on update of the current / previous resource in the api